### PR TITLE
[QA-AUTO] Automate "create board tile" component test

### PR DESCRIPTION
**Pull request by AgnieszkaTobolska**

### CODE REVIEW

**Review by ospatrycja — APPROVED** [review]

**Review by KlaudiaLapcionek — APPROVED** [review]
lgtm :)

File: src/test/java/com/intive/template/pages/HomePage.java [hunk 1]
```diff
@@ -21,6 +25,12 @@
     @FindBy(how = How.XPATH, using = "/html/body/div/div/header/div/a")
     WebElement logo;
 
+    @FindBy(how = How.XPATH, using = "//div[@role='button']")
```
Comment (MarekPawlowskiQA) [inline src/test/java/com/intive/template/pages/HomePage.java:28]: Could be risky later, when there will be more buttons.

**Review by MarekPawlowskiQA — COMMENTED** [review]
I would change what Patrycja mentioned and we are good to go

File: src/test/java/com/intive/template/pages/HomePage.java [hunk 2]
```diff
@@ -57,6 +67,26 @@ public void clickOnLogo() {
         logo.click();
     }
 
+    public void hoverCreateBoardTile() {
+        Actions hover = new Actions(driver);
+        hover.moveToElement(createBoardTile).perform();
+    }
+
+    public void createBoardTileClickable() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5,1));
```
[the remark this answers is not in the document]
Comment (AgnieszkaTobolska) [reply src/test/java/com/intive/template/pages/HomePage.java]: done

File: src/test/java/com/intive/template/pages/HomePage.java [hunk 3]
```diff
@@ -57,6 +67,26 @@ public void clickOnLogo() {
...
+        wait.until(ExpectedConditions.elementToBeClickable (createBoardTile));
```
[the remark this answers is not in the document]
Comment (AgnieszkaTobolska) [reply src/test/java/com/intive/template/pages/HomePage.java]: done

**Review by MarekPawlowskiAuto — APPROVED** [review]

**Review by MarekPawlowskiQA — APPROVED** [review]

**Pull request merged**